client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/TableFilterStyles.ts

```diff
@@ -71,7 +71,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     chip: {
         maxWidth: '100%',
         whiteSpace: 'nowrap',
-        overflow: 'hidden',
+        //overflow: 'hidden',
```

Comment:
you can just delete the line.
But don't it makes the input box to get bigger now? (when choosing several options)

Reply:
I will delete it.
The input remains in the same width, but I well check this once again.

Reply:
![image](https://user-images.githubusercontent.com/55889135/121157396-3232de00-c852-11eb-8ad6-1cfac3843b24.png)
I run it and it looks like that, is there a way to fix this?
If not I think it is still Ok because it only looks like that when we open the select dropdown. 👍🏼